src/main/scala/chisel3/iotesters/VCSBackend.scala

```diff
@@ -136,13 +136,13 @@ private[iotesters] object setupVCSBackend {
         The following block adds an annotation that tells the black box helper where the
         current build directory is, so that it can copy verilog resource files into the right place
          */
-        val annotationMap = firrtl.AnnotationMap(optionsManager.firrtlOptions.annotations ++ List(
+        val annotations = optionsManager.firrtlOptions.annotations ++ List(
           firrtl.annotations.Annotation(
             CircuitName(circuit.name),
             classOf[BlackBoxSourceHelper],
-            BlackBoxTargetDir(optionsManager.targetDirName).serialize
+            BlackBoxTargetDirAnno(optionsManager.targetDirName).serialize
           )
-        ))
```

Comment:
Instead of using `firrtl.annotations.Annotation(....)` (which is deprecated), you can just use the BlackBoxTargetDirAnno, ie.

```scala
val annotations = optionsManager.firrtlOptions.annotations ++
  List(BlackBoxTargetDirAnno(optionsManager.targetDirName))
```